package.json

```diff
@@ -95,6 +95,7 @@
         "react-router-dom": "^6.11.2",
         "react-stately": "^3.22.0",
         "tailwind-merge": "^1.14.0",
+        "usehooks-ts": "^2.14.0",
```

Comment:
This is no longer needed right?

Reply:
my bad, removed here [05ca45e](https://github.com/powerhouse-inc/document-model-electron/pull/93/commits/05ca45ef3227c50a7d44bfd7c8d8730a89d3c369)